build-support/githooks/pre-commit

```diff
@@ -52,15 +52,8 @@ echo "* Checking shell scripts via our custom linter"
 # When travis builds a tag, it does so in a shallow clone without master fetched, which
 # fails in pants changed.
 if git rev-parse --verify "${MERGE_BASE}" &>/dev/null; then
-  echo "* Checking formatting of Python files"
-  ./v2 --changed-parent="${MERGE_BASE}" lint || die "To fix formatting, run \`./v2 --changed-since=master fmt\`"
-
-  # TODO(CMLivingston) Make lint use `-q` option again after addressing proper workunit labeling:
-  # https://github.com/pantsbuild/pants/issues/6633
-  # TODO: add a test case for this while including a pexrc file, as python checkstyle currently fails
-  # quite often with a pexrc available.
-  echo "* Checking lint"
-  ./pants --exclude-target-regexp='testprojects/.*' --changed-parent="${MERGE_BASE}" lint || exit 1
+  echo "* Checking formatting and Flake8"
+  ./v2 --changed-parent="${MERGE_BASE}" lint || die "If there were formatting errors, run \`./v2 --changed-since=master fmt\`"
```

Comment:
Consistently using `parent` or `since` would be good.

Reply:
I don’t think we plan to make many changes to the plugin, do we?

Although, actually, maybe we end up super-charging the plugin to do validation for writing rules and plugins? That would be neat! Currently, Pants proactively does some of that every time it gets invoked, but maybe that validation should move into the plugin and we could expand the checks? 

(If we did that, though, it would be through the optional plugin pattern we created for the plugin. You must explicitly opt in to using those rule lints.)

Reply:
I agree. Will forward fix.